bot/tests/sending_videos/test_compile_selected_clips.py

```diff
@@ -8,72 +8,82 @@
 @pytest.mark.usefixtures("db_pool", "telegram_client")
 class TestMergeClipsCommand(BaseTest):
 
-    @pytest.mark.quick
+    @pytest.mark.asyncio
     async def test_merge_multiple_clips(self):
-        await self.send_command('/klip geniusz')
-        await  self.expect_command_result_contains(
-            '/zapisz klip1',
-            [save_msg.get_clip_saved_successfully_message("klip1")],
-        )
-        await  self.send_command('/klip kozioł')
+        clip1_name = "klip1"
+        clip2_name = "klip2"
+        clip3_name = "klip3"
```

Comment:
nie odpalalem tego w ide tylko czytam w webowce, ale jak widze cos takiego to zolte swiatlo zeby rozwazyc fora

Reply:
![image](https://github.com/user-attachments/assets/5246a6ce-2bea-4e96-8528-34eeefb1eef2)
może nie ma co kombinować